misc/iitthesis.cls

```diff
@@ -170,12 +170,42 @@
 \setmainlanguage{english} % Palestine was under British rule after all
 \IfFileExists{bidi.sty}{}{\ClassError{\filename}{The bidi package is not available for loading}{}}
 \AtEndPreamble{
- \setotherlanguage[numerals=arabic,calendar=gregorian]{hebrew}
+ \setotherlanguage[numerals=arabic,calendar=gregorian,fullyear=true]{hebrew}
  \newfontfamily\hebrewfont[Script=Hebrew]{David CLM}
  % These next two lines fix an issue with equation numbering parentheses;
  % see: https://tex.stackexchange.com/a/141437/5640
  \def\maketag@@@#1{\hbox{\m@th\normalfont\LRE{#1}}}
  \def\tagform@#1{\maketag@@@{(\ignorespaces#1\unskip)}}
+ \RequirePackage{hebrewcal}
+
+ % override hebrewcal.tex macros to remove the day of the month:
```

Comment:
Wrap this in a specific command, rather than making global changes, e.g. something like:
```
\newcommand{\thismonth}[1]{%
\makeatletter
 \def\@FormatForHebrew#1#2#3{%
  \HebrewMonthName{#2}{#3}~%
  \HebrewYearName{#3}}
 \def\@FormatForEnglish#1#2#3{%
  \HebrewMonthNameInEnglish{#2}{#3}~%
  \number#3}%
\makeatother
\today
}
```

Reply:
Also see my comment regarding the month and year package options. Since we likely need to retain them, What we actually need is to set their default values to `\today`'s month and year. (in Hebrew and in English).